Core/include/Acts/Propagator/EigenStepper.hpp

```diff
@@ -420,6 +422,11 @@ class EigenStepper {
   /// Overstep limit
   double m_overstepLimit;
 };
+
+template <typename navigator_t>
+struct supportsBoundParameters<EigenStepper<>, navigator_t>
```

Comment:
just noticed the starting lower-case. is this what we do with traits?

Reply:
Maybe not. I'll change it.